pages/your_tasks/data_brokering.md

```diff
@@ -0,0 +1,89 @@
+### Description
+Often, a goal of the broker is to submit the collected and harmonised data in an open archive for the benefit of both the data owners and the wider scientific community. This supports sharing data with collaborators, referencing for publication and provides a [long-term storage solution](storage).
+
+### Example for genomic data
+For sharing of sequencing data within Europe, the [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/about) can be used. The ENA is part of the [INSDC](https://www.insdc.org/), a global collaboration for the archival of nucleotide sequence data including NCBI and DDBJ. For information on the structure of ENA content, please see here: https://www.ebi.ac.uk/ena/browser/about/content. A general guide to submissions to the ENA can be found here: https://ena-docs.readthedocs.io/en/latest/submit/general-guide.html
+
+In the particular case of SARS-CoV-2 genomic data, more information will be available on the SARS-CoV-2 domain page (under construction, coming soon).
+
+### Considerations
+* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. More information can be found on our [data publication](data_publication) page.
+* If submitting to open access repositories, ensure clarity on which fields and data to share and, if these fields contain sensitive information, should they be anonymised/pseudonymised
+  * metadata anonymisation:  [sensitive data](sensitive_data), and [human data](human_data)
+  * Need to remove any human data (e.g. reads) prior to submission
+* Submitted data may not be deleted, but may be suppressed from public view upon request
+* Submitted records may be updated at a later date with new or amended metadata entries. However, it should be noted that upon creation of a new record, data is tagged for distribution and selected metadata fields are exchanged with other repositories. Redistribution of updated records is usually not triggered automatically, so updating records fully can be a time consuming and manual process.
+* After data is submitted to a public repository, should the original copy of the data be retained at the central brokering platform and linked to its public counterpart? Or should it be removed and replaced with the ID of the public record?
+* Within INSDC databases, data may be submitted and held privately for a period of time by setting a release date at time of submission, see [ENA documentation](https://ena-docs.readthedocs.io/en/latest/faq/release.html).
+* By default, data submitted using default accounts are accredited to the submitting institute. However, broker accounts have the ability to credit different institutes with different records: https://ena-docs.readthedocs.io/en/latest/faq/data_brokering.html 
```

Comment:
Dear @floradanna 
Thanks for your comments. Requesting a broker account is indeed the starting point, and this implies different things and comes with different rights depending where you're submitting as a broker, e.g. to ENA or GISAID. We can update that section accordingly and perhaps merge part of this section with the Data publication page where this is relevant both for individual or broker submissions.

Regarding the SARS-CoV-2 domain page, it will cover many more aspects than just brokering, like metadata standards specific to SARS-CoV-2, legal aspects, technical infrastructures and dedicated tools in place... so definitely worth it, we see it as the page where we will document all the lessons learnt within ELIXIR-CONVERGE WP9.

Reply:
Thanks @floradanna for reviewing. We have added a section to the 'Data Publication' page https://github.com/elixir-europe/rdmkit/pull/942 Kindly have a look, I appreciate your time.